browser/src/control/Control.Notebookbar.js

```diff
@@ -341,6 +342,14 @@ L.Control.Notebookbar = L.Control.extend({
 		this.reloadShortcutsBar();
 	},
 
+	disableNotebookbarControl: function(e) {
+		var toolbar = L.DomUtil.get('toolbar-up');
+		this.builder.executeAction(toolbar, {
```

Comment:
just `this.parentContainer` can be used

Reply:
It does not work, It looks like this.parentContainer was not updated when detach childs,
but it is another issue to address.